src/components/StandardCard/StandardCard.jsx

```diff
@@ -128,22 +129,37 @@ const StandardCard = ({ card, cards, setCards, ...props }) => {
           {card.criteria}
         </Text>
 
-        <HStack mt="auto" position="relative" mb="0.5">
-          {card.tags.slice(0, 3).map((tag, index) => {
-            return (
-              <Tag
-                key={index}
-                textTransform="capitalize"
-                bgColor="#C4D600"
-                rounded="14.7877px"
-                px="2"
-              >
-                {tag}
-              </Tag>
-            );
-          })}
+        <HStack
+          mt="auto"
+          mb="0.5"
+          display="flex"
+          justifyContent="space-between"
+        >
+          <Flex overflowX="hidden">
+            {card.tags.map((tag, index) => {
+              if (index < 3) {
```

Comment:
This part isn't working for me.
<img width="400" alt="image" src="https://user-images.githubusercontent.com/69729390/227806927-d0df102a-dcb0-42a2-95b6-3e5b5bde5e48.png">

I fixed it a little bit adding a scroll property. I'm not sure honestly what the designers want so honestly don't worry about this one